src/Microsoft.OpenApi/Models/OpenApiCallback.cs

```diff
@@ -45,10 +45,10 @@ public class OpenApiCallback : IOpenApiSerializable, IOpenApiReferenceable, IOpe
         /// </summary>
         public OpenApiCallback(OpenApiCallback callback)
         {
-            PathItems = new(callback.PathItems);
-            UnresolvedReference = callback.UnresolvedReference;
-            Reference = new(callback.Reference);
-            Extensions = new Dictionary<string, IOpenApiExtension>(callback.Extensions);
+            PathItems = new(callback?.PathItems);
+            UnresolvedReference = callback?.UnresolvedReference ?? false;
```

Comment:
```suggestion
            UnresolvedReference = callback?.UnresolvedReference ?? UnresolvedReference;
```

(and repeat for booleans)
Booleans could have a default value, and we don't want to override with a new default